### PR TITLE
RBTA-95-bug-report-edit-mode-input-fields-kiparisova

### PR DESCRIPTION
**Pull request by Kiparisova**
https://jira.pasv.us/browse/RBTA-95

### CODE REVIEW

**Comment by Kiparisova** [comment]
@itrusevich Thank you for the suggestions. I have revised it. Please review again.

File: test/specs/reg/bug-report/edit-mode-.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
+import {assert} from 'chai';
+import sel from '../../../selectors/bug-report';
+import exp from '../../../expected/bug-report';
+import help from '../../../helpers/helpers';
+
+describe('Summary Input Field Properties', function () {
+
+    it('Font Family', function () {
+        browser.url('/');
```
Comment (artsenius) [inline test/specs/reg/bug-report/edit-mode-.js]: Seem like it's present in login, so you can remove it

Comment (Kiparisova) [reply test/specs/reg/bug-report/edit-mode-.js]: 1.Got read of excessive variables and browser.url('/'); 
2.Added tests for verification Single line input;
3.Resolved conflicts.
Please review again.

File: test/specs/reg/bug-report/edit-mode-.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import help from '../../../helpers/helpers';
 
 describe('Summary Input Field Properties', function () {
 
-    it('Summary Input Field is a text filed and accepts only one line', function () {
+    it('Summary Input Field is a text field and accepts only one line', function () {
```
Comment (Dmitricas86) [inline test/specs/reg/bug-report/edit-mode-.js]: Good catch @itrusevich .
Didn't notice this typo

**Pull request merged**